eventorganizer.api/src/main/java/com/event/organizer/api/model/dto/FeedbackRequestDto.java

```diff
@@ -0,0 +1,33 @@
+package com.event.organizer.api.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class FeedbackRequestDto {
+    private final Integer min = 1;
```

Comment:
MIN and MAX should be constants 
private static final Integer MIN = 1;
private static final Integer MAX = 5;